### DISCUSSION TITLE
Is it possible to disable SSL certification verification?

### DISCUSSION THREAD

**Discussion by eroji** [provectus/kafka-ui#4459] [General]
I have a bunch of internal Kafka clusters with SASL_SSL authentication required that I'm trying to get kafka-ui to connect to. Each of which has its own set of self-signed certificates. Is there a way to disable certificate validation in the UI or configuration?

**Comment by germanosin** [comment]
Me and @Haarolean no longer support this repo.
Don't you mind to propose your request to our repo https://github.com/kafbat/kafka-ui
